projects/web/src/components/epcis/common.tsx

```diff
@@ -4,6 +4,14 @@ import * as QrReader from "react-qr-reader";
 import { Event } from "../../epcis";
 const { DigitalLink } = require("digital-link.js");
 
+import { AuthState } from "../../auth";
+import { Organisation } from "../../business-registry";
+
+export interface Props {
```

Comment:
My preference is to leave the Props type def right above the react component as the two are really closely linked.

I understand that in this case there is a small amount of code dupe, but since it's only a type def and in most cases the props you pass to a component will vary, I don't see the need to extract it out.